Added VS-Dark theme

Twineee1 opened the pull request:
<!-- ie, `Resolves #1234`, etc... so GitHub can magically link it -->
Resolves #4271 

### Changes
I edited the base vs theme the match VS dark

### Checklist
- [x] Added VSDark theme

joshgoebel commented:
Thanks! - but I believe consistent naming would be `vs-dark` (with a dash).  Rename and I think we can merge! (add a CHANGES.md entry also please).

Twineee1 commented:
> Thanks! - but I believe consistent naming would be `vs-dark` (with a dash). Rename and I think we can merge! (add a CHANGES.md entry also please).

I renamed and updated it.

Twineee1 commented:
🥳 first PR!
